services/ows.py

```diff
@@ -1844,6 +1844,326 @@
                 "default_style": "dry_observations",
             }
         ]
+    },
+        {
+        # Name and title of the platform layer.
+        # Platform layers are not mappable. The name is for internal server use only.
+        "name": "WOfS Collection2",
+        "title": "Water Observations from Space c2",
+        "abstract": "WOfS",
+        # Products available for this platform.
+        # For each product, the "name" is the Datacube name, and the label is used
+        # to describe the label to end-users.
+        "products": [
+            {
+                # Included as a keyword  for the layer
+                "label": "WOfS C2 Statistics",
+                # Included as a keyword  for the layer
+                "type": "Water Summary",
+                # Included as a keyword  for the layer
+                "variant": "30m",
+                # The WMS name for the layer
+                "name": "wofs_2_annual_summary_statistics",
+                # The Datacube name for the associated data product
+                "product_name": "ga_ls8c_wofs_2_annual_summary",
+                "abstract": """Water Observations from Space (WOfS) Statistics is a set of statistical summaries of 
+                the WOfS product which combines WOfS observations into summary products that help the understanding 
+                of surface water across Africa. WOfS Statistics is calculated from the full depth time series (1984 ? 
+                2018). The water detected for each location is summed through time and then compared to the number of 
+                clear observations of that location. The result is a percentage value of the number of times water 
+                was observed at the location. The layers available are: the count of dry observations; the count of 
```

Comment:
there are no dry observations, there is a count of clear observation (no cloud/shadow)